canopeum_backend/canopeum_backend/models.py

```diff
@@ -150,32 +189,59 @@ def upload_to(_, filename):
 class Asset(models.Model):
     asset = models.FileField(upload_to=upload_to, null=False)
 
+    def delete(self, using=None, keep_parents=False):
```

Comment:
Is `keep_parents` there because it's an overload? If so:
`from typing import overload`
```suggestion
    @overload
    def delete(self, using=None, keep_parents=False):
```